### PR TITLE
Cosmetics

### PR DESCRIPTION
**Pull request by framiere**
trapping Control-C
rename crash.log to conduktor-platform.log
use info instead of print

### CODE REVIEW

File: example-local/autorun/autorun.sh [hunk 1]
```diff
@@ -92,8 +94,8 @@ function prune() {
 
     info "Cleaning up docker images (use NO_PRUNE=true to prevent this)"
     for image in ${COMPOSE_IMAGES}; do
-      printf "Pruning docker image %s..." "${image}"
-      docker image rm "${image}" > /dev/null && echo " OK" || echo "KO. Error pruning, skipping..."
+      info "Pruning docker image %s..." "${image}"
```
Comment (sderosiaux) [inline example-local/autorun/autorun.sh:97]: printf was on purpose, don't like it?

Comment (sderosiaux) [inline example-local/autorun/autorun.sh:97]:
```suggestion
      printf "Pruning docker image %s..." "${image}"
```

Comment (framiere) [reply example-local/autorun/autorun.sh]: fixed.

**Review by sderosiaux — APPROVED** [review]

**Comment by sderosiaux** [comment]
does not work for me (M1), i have error 2, not 130

**Comment by framiere** [comment]
I have a m1 too :/ 
```
[+] Running 3/4
 ⠿ Container tmpbvzp5x3u-conduktor-platform-1  Stopped 1.1ss
 ⠿ Container tmpbvzp5x3u-schema-registry-1     Stopped 10.4ss
 ⠿ Container tmpbvzp5x3u-kafka-1               Stopped 1.4ss
 ⠦ Container tmpbvzp5x3u-zookeeper-1           Stopping 0.6s
canceled

* -> Conduktor Platform stopped by CTRL+C

[+] Running 9/9
 ⠿ Container tmpbvzp5x3u-conduktor-platform-1  Removed 0.0s
 ⠿ Container tmpbvzp5x3u-schema-registry-1     Removed 0.0s
 ⠿ Container tmpbvzp5x3u-kafka-1               Removed 0.0s
 ⠿ Container tmpbvzp5x3u-zookeeper-1           Removed 0.0s
 ⠿ Volume tmpbvzp5x3u_zookeeper_data           Removed 0.0s
 ⠿ Volume tmpbvzp5x3u_kafka_data               Removed 0.0s
 ⠿ Volume tmpbvzp5x3u_conduktor_data           Removed 0.1ss
 ⠿ Volume tmpbvzp5x3u_zookeeper_datalog        Removed 0.0s
 ⠿ Network tmpbvzp5x3u_default                 Removed 0.1ss

* Cleaning up docker images (use NO_PRUNE=true to prevent this)

Pruning docker image conduktor/conduktor-platform... OK
Pruning docker image conduktor/kafka:3.2.0... OK

```

**Comment by sderosiaux** [comment]
I didn't have the latest Docker Desktop version, from 1.29.2 to 2.10.2 now. It's indeed quite different and working.

**Pull request merged**